kfdefs/base/monitoring/blackbox-exporter/superset-blackbox-service-monitor.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: superset-blackbox-service-monitor
+spec:
+  endpoints:
+    - interval: 30s
+      metricRelabelings:
+        - sourceLabels:
+            - __address__
+          targetLabel: __param_target
+        - sourceLabels:
+            - __param_target
+          targetLabel: instance
+        - replacement: 'https://superset.datahub.redhat.com/'
+          targetLabel: target
+      params:
+        module:
+          - http_2xx
+        target:
+          - 'https://superset.datahub.redhat.com/'
+      path: /probe
+      targetPort: 9115
+  jobLabel: prometheus.io/joblabel
+  namespaceSelector:
+    matchNames:
+      - opf-jupyterhub
+  selector:
+    matchLabels:
+      prometheus.io/scrape: 'true'
```

Comment:
So this servicemonitor just tells prometheus to scrape the services with `prometheus.io/scrape: 'true'` label in them.